tests/test_exceptions.py

```diff
@@ -262,7 +252,7 @@ def tempest(_):
 
 
 def test_exception_in_ws_logged(caplog):
-    app = Sanic(__file__)
+    app = Sanic(__name__)
```

Comment:
Seems to be the only app name you haven't changed to a fixed string. Missed it or intentional?

Reply:
Should be fine here. The problem was the examples are actually used in some tests (which should really not be the case). So, their `__name__` includes a dot because of the import path.